The job **`aarch64-gnu-llvm-19-1`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/17250811913/job/48952400003) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/48952400003) [(plain)](https://github.com/rust-lang/rust/commit/adddae65368c28d36c4202458aa9bf387c5a6d06/checks/48952400003/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
test [crashes] tests/crashes/137467-2.rs ... ok
test [crashes] tests/crashes/137468.rs ... ok
test [crashes] tests/crashes/137467-3.rs ... ok
test [crashes] tests/crashes/137514.rs ... ok
2025-08-26T21:59:37.989626Z ERROR compiletest::runtest: fatal error, panic: "crashtest no longer crashes/triggers ICE, hooray! Please give it a meaningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR description ensures that the corresponding ticket is auto-closed upon merge. If you want to see verbose output, set `COMPILETEST_VERBOSE_CRASHES=1`."
test [crashes] tests/crashes/137580.rs ... FAILED
test [crashes] tests/crashes/137888.rs ... ok
test [crashes] tests/crashes/137582.rs ... ok
test [crashes] tests/crashes/138008.rs ... ok
test [crashes] tests/crashes/138009.rs ... ok
---

---- [crashes] tests/crashes/137580.rs stdout ----
------rustc stdout------------------------------

------rustc stderr------------------------------
error: argument never used
##[error] --> /checkout/tests/crashes/137580.rs:3:25
  |
3 |     println!("%65536$", 1);
  |                         ^ argument never used
  |
note: format specifiers use curly braces, and the conversion specifier `
      ` is unknown or unsupported
 --> /checkout/tests/crashes/137580.rs:3:15
  |
3 |     println!("%65536$", 1);
  |               ^^^^^^^^
  = note: printf formatting is not supported; see the documentation for `std::fmt`

error: aborting due to 1 previous error


------------------------------------------

error: crashtest no longer crashes/triggers ICE, hooray! Please give it a meaningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR description ensures that the corresponding ticket is auto-closed upon merge. If you want to see verbose output, set `COMPILETEST_VERBOSE_CRASHES=1`.

thread '[crashes] tests/crashes/137580.rs' panicked at src/tools/compiletest/src/runtest/crashes.rs:17:18:
fatal error
stack backtrace:
   0: __rustc::rust_begin_unwind
```

</details>

--- a/tests/crashes/137580.rs
+++ b/tests/crashes/137580.rs
@@ -1,4 +1,0 @@
-//@ known-bug: #137580
-fn main() {
-    println!("%65536$", 1);
-}

--- a/tests/ui/macros/format-foreign-dollar-without-spec.rs
+++ b/tests/ui/macros/format-foreign-dollar-without-spec.rs
@@ -1,3 +1,4 @@
+// https://github.com/rust-lang/rust/issues/137580
 fn main() {
     println!("%65536$", 1);
     //~^ ERROR never used

--- a/tests/ui/macros/format-foreign-dollar-without-spec.stderr
+++ b/tests/ui/macros/format-foreign-dollar-without-spec.stderr
@@ -1,12 +1,12 @@
 error: argument never used
-  --> $DIR/format-foreign-dollar-without-spec.rs:2:25
+  --> $DIR/format-foreign-dollar-without-spec.rs:3:25
    |
 LL |     println!("%65536$", 1);
    |                         ^ argument never used
    |
 note: format specifiers use curly braces, and the conversion specifier `
       ` is unknown or unsupported
-  --> $DIR/format-foreign-dollar-without-spec.rs:2:15
+  --> $DIR/format-foreign-dollar-without-spec.rs:3:15
    |
 LL |     println!("%65536$", 1);
    |               ^^^^^^^^

